### PR TITLE
Update snirf_specification.md acknowledgements

### PR DESCRIPTION
**Pull request by dboas**
Qianqian, please check the formatting of the acknowledgements and fix if needed. thanks

### CODE REVIEW

**Comment by fangq** [comment]
hi David, I am going to close this pull request. I think it was pull towards an incorrect repo.

Please feel free to directly commit your change to fNIRS/snirf, which is the "official repository" of this specification. 

However, if you want to add a layer of review of your changes, what you should do, for future updates, is that you 

1. first fork fNIRS/snirf to dboas/snirf, 
2. make changes in your fork and commit the changes;
3. click on the "New Pull Request" in dboas/snirf,then click on the link "compare across forks" on the top
4. in "base", select "fNIRS/snirf", and in "head fork", select "dboas/snirf", then create the pull request, you can assign me or jay or someone else as the reviewer.

once this pull request is sent to fNIRS/snirf, I/Jay will take a look, and if looks good, will merge the change. 

In other words, we all make changes in our own forks (fangq/snirs, dboas/snirf, and jayd1860/snirf) and then send pull request towards the official repo - fNIRS/snirf.

if it sound a bit complicated, again feel free to directly change fNIRS/snirf.

**Comment by dboas** [comment]
Qianqian, thanks for the guidance on this. I will only get it once I try it a few times. Next time I will ask a local person to sow me the way. I think Nathan knows this stuff.
